.dockerignore

```diff
@@ -292,7 +292,7 @@
 !**/src/pal/prebuilt/idl/*_i.c
 
 # Valid 'debug' folder, that contains CLR debugging code
-!**/src/debug
+!**/src/**/debug
```

Comment:
Is this going to hide https://github.com/dotnet/runtime/tree/main/src/coreclr/debug ?

Reply:
It's the other way around: the `!` syntax is adding an exception for that directory, so it's not filtered.

Reply:
For some reason stuff containing the word `debug` seems to be ignored by default. (No idea why, can't see it in `.dockerignore` + I'm new to docker)

Reply:
I think the culprit is this:
https://github.com/dotnet/runtime/blob/93de12767f79f8e5e0e9a73cb6b0f3470372a35d/.dockerignore#L17